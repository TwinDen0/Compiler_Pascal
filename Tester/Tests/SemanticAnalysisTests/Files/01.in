var
a : string;
b : integer;
c,a : real;
begin
end.