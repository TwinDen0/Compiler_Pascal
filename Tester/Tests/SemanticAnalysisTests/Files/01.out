(4,3) Fatal: Duplicate identifier "a"